cypress/start.sh

```diff
@@ -46,9 +46,9 @@ fi
 
 
 RETVAL=$?
-echo printing $RETVAL
+echo "printing $RETVAL"
 
-if [ $RETVAL -eq 0 ]; then
+if [[ $RETVAL -eq 0 ]]; then
```

Comment:
did that not work before then? bash syntax is weird ;)

Reply:
I got an error about "unary operator" with the other syntax. I checked the above works in both bash and zsh.